cvmfs/catalog_mgr_impl.h

```diff
@@ -1,3 +1,13 @@
+/*
+ * catalog_mgr_impl.h
+ *
+ *  Created on: Aug 13, 2015
+ *      Author: molina
+ */
+
+#ifndef CVMFS_CATALOG_MGR_IMPL_H_
+#define CVMFS_CATALOG_MGR_IMPL_H_
+
 /**
```

Comment:
It appears correctly in the full changes. I don't know why in the conversation it still appear like in the old version